<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RicercaAcquisto</class>
 <widget class="QWidget" name="RicercaAcquisto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RicercaAcquisto</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Digita codice acquisto che vuoi ricercare:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Codice:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_codice">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cerca">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerca</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_apri">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Apri</string>
   </property>
  </widget>
  <widget class="QListView" name="lista_acquisti">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>481</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
